app/src/main/res/layout/activity_spiral.xml

```diff
@@ -69,14 +69,15 @@
         android:layout_marginTop="427dp" />
 
     <TextView
-        android:text="@string/spiral_info"
+        android:text="Trace spiral"
```

Comment:
Why change this to be a hard coded string instead of a constant from `strings.xml`?